src/fontcache.h

```diff
@@ -206,6 +208,11 @@ struct FontCacheSettings {
 	bool global_aa;             ///< Whether to anti alias all font sizes.
 };
 
+const int DEFAULT_FONT_MAX_HEIGHT = 40; ///< The maximum height allowed by font sliders.
+const int DEFAULT_FONT_MIN_MAX_GAP = 14; ///< This controls how many steps will be on any slider. The gap between the low end of the slider and the high end.
```

Comment:
There's a FONT_SLIDER_MARK_COUNT in the settings_gui.cpp file.  Honestly since this variable is just used to construct that one it's kind of redundant. Removed.